core/new-gui/package.json

```diff
@@ -81,24 +79,23 @@
     "tinyqueue": "2.0.3",
     "tslib": "2.3.1",
     "uuid": "8.3.2",
+    "vscode": "npm:@codingame/monaco-vscode-api@~1.76.6",
     "vscode-ws-jsonrpc": "3.1.0",
     "y-monaco": "0.1.5",
     "y-protocols": "1.0.5",
     "y-quill": "0.1.5",
     "y-websocket": "1.4.0",
     "yjs": "13.5.41",
-    "zone.js": "0.14.4",
-    "@babel/helper-replace-supers": "~7.18.6"
+    "zone.js": "0.13.0"
```

Comment:
Why downgrade zone?

Reply:
It's the matched version for Angular 16.